app/plugins/std_workflow/js/std_workflow_entity_shared_role.js

```diff
@@ -29,6 +31,17 @@ var tableSharedRolesSelect = function(M, entityName) {
     return q.length ? q[0] : null;
 };
 
+var replaceActionableByMaybe = function(M, actionableBy) {
+    if(USE_TARGET_FOR_ENTITIES.length) {
```

Comment:
I think this will throw an exception is the config isn't defined. You should default it to [] on line 14 above to avoid this